src/app/view-modes/image-image/image-image.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { DisplayGrid, GridsterConfig, GridsterItem, GridType } from 'angular-gridster2';
+
+@Component({
+  selector: 'evt-image-image',
+  templateUrl: './image-image.component.html',
+  styleUrls: ['./image-image.component.scss'],
+})
+export class ImageImageComponent implements OnInit {
+  public layoutOptions: GridsterConfig = {};
+  public imagePanel1Item: GridsterItem = { cols: 1, rows: 1, y: 0, x: 0 };
+  public imagePanel2Item: GridsterItem = { cols: 1, rows: 1, y: 0, x: 1 };
+
+  ngOnInit() {
+    this.initGridster();
+  }
+
+  private initGridster() {
+    this.layoutOptions = {
```

Comment:
you can put this at line 10.

Reply:
If you move it at line 15 (15 is the new 10 XD), you can remove the method `initGridster`